src/ui/components/AccordionItem.tsx

```diff
@@ -15,13 +15,14 @@ import { v4 as uuidv4 } from 'uuid'
 
 import { AccessibilityRole } from 'libs/accessibilityRole/accessibilityRole'
 import { useFunctionOnce } from 'libs/hooks'
+import FilterSwitch, { FilterSwitchProps } from 'ui/components/FilterSwitch'
 import { TouchableOpacity } from 'ui/components/TouchableOpacity'
 import { touchableFocusOutline } from 'ui/theme/customFocusOutline/touchableFocusOutline'
 import { getHeadingAttrs } from 'ui/theme/typographyAttrs/getHeadingAttrs'
 import { useCustomSafeInsets } from 'ui/theme/useCustomSafeInsets'
 
 import { ArrowNext as DefaultArrowNext } from '../svg/icons/ArrowNext'
-import { getSpacing, Typo } from '../theme'
+import { getSpacing, Spacer, Typo } from '../theme'
 
 interface IAccordionItemProps {
```

Comment:
je sais que tu ne l'as pas changé, ce `I` ne sert à rien

```suggestion
interface AccordionItemProps {
```